tracecat/cli/workflow.py

```diff
@@ -123,6 +123,20 @@ async def _get_cases(workflow_id: str):
     return res.json()
 
 
+async def _create_activate_workflow(
+    title: str | None = None,
+    description: str | None = None,
+) -> str:
+    """Create and activate a workflow with a webhook.
+
+    Mainly used for testing.
+    """
+    workflow = await _create_workflow(title, description)
+    workflow_id = workflow["id"]
+    await _activate_workflow(workflow_id, with_webhook=True)
+    return workflow_id
+
+
```

Comment:
if this is only used for testing, let's put it in the test file

Reply:
Might be used else where. It's useful enough not to be just a testing thing imo. Just added that comment to remind me what's it for (for now)